.github/workflows/build_and_test.yml

```diff
@@ -872,12 +882,17 @@ jobs:
         cd OpenDDS/ACE_TAO
         export ACE_COMMIT=$(git rev-parse HEAD)
         echo "ACE_COMMIT=$ACE_COMMIT" >> $GITHUB_ENV
+    - name: get Compiler version
+      shell: bash
+      run: |
+        export COMPILER_VERSION=$("c++ --version 2>&1 | head -n 1")
```

Comment:
This build is using installing and using clang, so `c++` isn't going to be the right compiler.

Reply:
Ok, I'll change it to "clang++ --version 2>&1 | head -n 1"
That will be the same for these two jobs, right?
1) u18_clang5_i0w1_sec
2) u18_clang10_sec_js0

Reply:
You'll want to explicitly add the version to the executable or you'll be getting a system default. For Ubuntu, I believe this looks like: clang++10, clang++9, clang++8, clang++7, clang++6.0, clang++5.0 (note the decimal in 6 and 5).

Reply:
replaced with "clang++-5.0 --version ..." or "clang++-10 --version ..."